dev/routing/users.js

```diff
@@ -40,7 +40,7 @@ module.exports = function (server) {
       },
       handler: function (request, reply) {
         let user = request.payload;
-        let serverUrl = request.headers.referer;
+        let serverUrl = request.headers.host;
```

Comment:
I received this link "localhost#/new_password/TCToNWt7kbRim7Ah" and it is not working